eodag/rest/stac.py

```diff
@@ -762,6 +762,7 @@ def get_collections(
             for collec in collections["collections"]
         ]
 
+        collections["stac_version"] = self.stac_config["stac_version"]
```

Comment:
```suggestion
        collections = format_dict_items(collections, **format_args)

```

Reply:
better to use `format_dict_items` here, like it is done elsewhere to format *curly brackets* variables